jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -424,12 +426,32 @@ private void executionQueue(Either<Execution, DeserializationException> either)
                         .filter(workerTask -> this.deduplicateWorkerTask(execution, executorState, workerTask.getTaskRun()))
                         .toList();
 
+
                     // WorkerTask not flowable to workerTask
                     workerTasksDedup
                         .stream()
-                        .filter(workerTask -> workerTask.getTask().isSendToWorkerTask())
+                        .filter(workerTask -> {
+                            try {
+                                return workerTask.getTask().isSendToWorkerTask() && TruthUtils.isTruthy(workerTask.getRunContext().render(workerTask.getTask().getRunIf()));
+                            } catch (IllegalVariableEvaluationException e) {
+                                return true;
```

Comment:
You should at least log a WARNING or ERROR.
I wonder if it would not be best to fail the task